src/InverseProblems.jl

```diff
@@ -391,14 +368,22 @@ function drop_y_dimension(grid::SingleColumnGrid)
 end
 
 #####
-##### VectorNormMap
+##### ConcatenatedVectorNormMap 
 #####
 
-struct VectorNormMap end
+"""
+    ConcatenatedVectorNormMap()
+
+Forward map transformation of simulation output to a scalar by
+taking a naive `norm` of the difference between concatenated vectors of the
+observations and simulation output.
+"""
+struct ConcatenatedVectorNormMap end
 
-observation_map(::VectorNormMap, observations) = reshape([0], 1, 1)
+output_map_str(::ConcatenatedVectorNormMap) = "ConcatenatedVectorNormMap"
+observation_map(::ConcatenatedVectorNormMap, observations) = reshape([0], 1, 1)
```

Comment:
a bit further down there is a redefinition of this method:

```julia
observation_map(map::ConcatenatedVectorNormMap, observations) = hcat(0.0)
```

which one we want?

Reply:
is it because of this that docs fail?

```
** incremental compilation may be fatally broken for this module **
```